src/utils/strings.ts

```diff
@@ -56,3 +56,4 @@ export const formatNumberForDisplay = (
 }
 
 export const DATE_TIME_FORMAT = 'DD/MM/YYYY HH:mm:ss [UTC]Z'
+export const SIMPLE_DATE_FORMAT = 'DD/MM/YYYY'
```

Comment:
nit: How about merging these?

```suggestion
export const SIMPLE_DATE_FORMAT = 'DD/MM/YYYY'
export const DATE_TIME_FORMAT = `${SIMPLE_DATE_FORMAT} HH:mm:ss [UTC]Z`
```

Reply:
Done!